onnx/test/shape_inference_test.py

```diff
@@ -24,12 +24,14 @@ def _make_graph(self,
             initializer = []
         names_in_initializer = set(x.name for x in initializer)
         input_value_infos = []
+        proto_type = TensorProto.FLOAT  # Should be set; checker does not allow UNDEFINED
```

Comment:
Better to move this into the else branch of the first if-else inside the for-loop below (line 35/37)